### DISCUSSION TITLE
Translation

### DISCUSSION THREAD

**Discussion by gkapelakos** [rafaelmardojai/blanket#274] [Ideas]
Hello I want to translate blanket to Greek. If anyone can guide me on how to do it i would be thankful!

**Comment by rafaelmardojai** [comment]
You can translate the app through Transifex https://www.transifex.com/rafaelmardojai/blanket/.

Just saw you request to create the language and approved. The rest should be more or less intuitive.

**Reply by gkapelakos** [reply]
I've finished the translation. Although the name Blanket cannot be changed cause it sounds super weird in Greek.

**Reply by rafaelmardojai** [reply]
Approved and pushed!

**Reply by gkapelakos** [reply]
Thanks!! I'll try it and report any bugs if there are any
